Manager/src/main/java/io/raspberrywallet/manager/common/readers/StringReader.java

```diff
@@ -0,0 +1,56 @@
+package io.raspberrywallet.manager.common.readers;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.Synchronized;
+
+import java.security.SecureRandom;
+
+public class StringReader extends Reader<String> {
```

Comment:
What is this class for ? And why is it called StringReader while it lazly write and returns random string ?

Reply:
Well it is not longer used, I've forgot to delete this. Will change